web-jehlomat/src/screens/SetNewPassword/SetNewPassword.tsx

```diff
@@ -0,0 +1,144 @@
+﻿import React, {FunctionComponent} from "react";
+import {useHistory} from "react-router-dom";
+import {useMediaQuery} from "@mui/material";
+import {Formik, FormikHelpers} from "formik";
+import {AxiosResponse} from "axios";
+import * as yup from "yup";
+import API from "config/baseURL";
+import {media} from "utils/media";
+import hasOwnProperty from "utils/hasOwnProperty";
+import SignInImage from "assets/images/sign-in.svg";
+import {Header} from "Components/Header/Header";
+import PrimaryButton from "Components/Buttons/PrimaryButton/PrimaryButton";
+import TwoColumns from "Components/Layout/TwoColumns";
+import {Input,Inputs,FormTitle,TransparentForm,Buttons,Illustration} from "./Components";
+
+const texts = {
+  TITLE: "Nové heslo",
+  MOBILE_TITLE: "Nové heslo",
+  INPUT__PASS__LABEL: "Nové heslo",
```

Comment:
why 2 underscores?

Reply:
It the suppose to be a delimiter to structure, or more likely group strings together.
`INPUT__` should point only to the inputs, then `PASS__` defines the given input and `LABEL` or `PLACEHOLDER` define target prop.
The `_` (single underscore) is to split two-word names (like the mobile title).

Reply:
thanks